composables/transaction/transactionBurn.ts

```diff
@@ -62,7 +62,7 @@ export function execBurnTx(item: ActionConsume, api, executeTransaction) {
         }
       )
       const cb = hasOffers.value
-        ? api.tx.utility.batch
+        ? api.tx.utility.batchAll
```

Comment:
I would leave batch here so nft is still burned

Reply:
I don't understand why you made this changes, I thought:
- `batch` => single tx call
- `batchAll` => multi tx call

Reply:
Oh... [nvm](https://substrate.stackexchange.com/questions/2084/what-is-the-difference-between-the-extrinsics-utility-batch-and-utility-batcha)

Reply:
So `batchAll` should be correct answer?
if withdraw doesn't work then burn won't work. no need to commit each tx

Reply:
>     * `batchAll` - all or nothing - so NFT won't be burned

This is the case we are looking for, not possible to burn nft with offer so I'd like to use `batchAll` in this particular case (`hasOffers.value`)

Reply:
If it throw error then I'm good with that.
`batch` doesn't throw error if a tx failed